contrib/runners/python_runner/python_runner/python_runner.py

```diff
@@ -322,7 +322,8 @@ def _get_output_values(self, exit_code, stdout, stderr, timed_out):
 
         if ACTION_OUTPUT_RESULT_DELIMITER in stdout:
             split = stdout.split(ACTION_OUTPUT_RESULT_DELIMITER)
-            assert len(split) == 3
+            if not len(split) == 3:
```

Comment:
Weird comparison here.

```suggestion
            if len(split) != 3:
```